go/extension/extension.go

```diff
@@ -141,6 +147,41 @@ func (e *FractalExt) GrantsFor(ctx *types.ExecutionContext, values ...*types.Sca
 	return encodeScalarValues(exist)
 }
 
+func (e *FractalExt) HexToBase58(ctx *types.ExecutionContext, values ...*types.ScalarValue) ([]*types.ScalarValue, error) {
+	inputHex, err := values[0].String()
+	if err != nil {
+		return nil, fmt.Errorf("convert value to string failed: %w", err)
+	}
+	binaryString, _ := hex.DecodeString(inputHex)
```

Comment:
I'd make an assertion here about the `len(binaryString)`.

Reply:
I would keep it as it is because in this case some wrong hex/binary string doesn't harm us - Kwil just won't find such a record.
In global i would describe all possible cases of what can come from user in `@caller` field and `$address` and `$public_key` fields and what is our logic to process it.

Reply:
Talked over voice, decided to do `if len(base58) != 64 || base58 == "" {`